dev-build/app/Global.scala

```diff
@@ -69,6 +69,7 @@ with ConfigAgentLifecycle
 with SurgingContentAgentLifecycle
 with SectionsLookUpLifecycle
 with ABTHeadlinesLifecycle
+with MostPopularFacebookAutoRefresh.Lifecycle
```

Comment:
We currently have 12 AutoRefreshes in the codebase, and 12 manually written classes that do what could be automatically generated. I'll update this pull request to remove them now ...